cfpb/consumerfinance.gov
# Restore two Ask CFPB Spanish translations

This PR corrects the following two translation regressions:

**1. "Read answer" on the landing page**

This translation was automatically commented out in the translation file for unknown reasons. I manually restored it.

**2. "Was this answer helpful to you?" on the feedback module**

It appears that this translation was automatically commented out due to the text being un-hard-coded in our templates and moved to a field in Wagtail. I manually restored it.

I'm not sure if either will get commented again when someone else runs `makemessages`. We really need to figure out a way to prevent that from happening in the future.


## How to test this PR

1. Pull branch
2. Open http://localhost:8000/es/obtener-respuestas/ and observe that "Read answer" links are now translated (compare to the untranslated string in production)
3. Go into an answer and observe that "Was this answer helpful to you?" is now translated (compare to the untranslated string in production)

Or wait for the dev Docker stack deployment and check it out there.


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
